common/changes/@itwin/desktop-viewer-react/integrate-unified-selection_2024-07-02-08-51.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/desktop-viewer-react",
+      "comment": "Integrated `@itwin/unified-selection` for unified selection handling when `SelectionStorage` is provided.",
```

Comment:
actually think you can set the comment to an empty string, and type to none.
you made the change in the viewer, and that should propagate up into these pkgs automatically

Reply:
I'm not sure I agree. `@itwin/viewer-react` is a direct dependency of `@itwin/desktop-viewer-react` and consumers are expected to consume `@itwin/desktop-viewer-react` directly, aren't they? In that case, the change in `@itwin/viewer-react` will be hidden from them, even though the API additions will become available in `@itwin/desktop-viewer-react`.

Reply:
True, the hope has always been users look at the changelog for viewer-react itself, but I see your point about it getting lost in the weeds. This is a minor nit so resolving